rl_studio/agents/mountain_car/settings.py

```diff
@@ -0,0 +1,45 @@
+###########################
+# Global variables file
+###########################
+from pydantic import BaseModel
+
+
+class QLearnConfig(BaseModel):
```

Comment:
This class has the same name as Qlearning. Shouldn't it be called `class MountainCarConfig`? (correct me if I'm wrong).